docs/pages/implementation/nuts-auth.rst

```diff
@@ -0,0 +1,225 @@
+
+The ``EN:PractitionerLogin:v3`` part defines the **language**, **type** and **version**. The dutch version of the given contract would be ``NL:BehandelaarLogin:v3``.
+
+You can draw up a contract by calling:
+
+.. code-block::
+
+    POST /internal/auth/experimental/contract/drawup HTTP/1.1
+    Host: server.example.com
+    Content-Type: application/json
+
+    {
+        "type": "PractitionerLogin",
+        "language": "EN",
+        "version": "v3",
+        "legalEntity": "urn:oid:2.16.840.1.113883.2.4.6.1:000000001",
+        "validFrom": "2019-06-24T14:32:00+02:00",
+        "validDuration": "2h"
+    }
+
+The ``legalEntity`` field contains the identifier of the care organization the user is currently representing. Please check the `Nuts Auth API`_ spec for the specifics.
+
+This will return a filled contract where the **legalEntity** has been transformed to the correct organization name:
+
+.. code-block::
+
+    {
+        "type": "PractitionerLogin",
+        "language": "EN",
+        "version": "v3",
+        "message": "EN:PractitionerLogin:v3 I hereby declare to act on behalf of Nursing home A. This declaration is valid from Monday, 24 June 2019 14:32:00 until Monday, 24 June 2019 16:32:00."
+    }
+
+The ``message`` part will have to used in the next step.
+
+User signature
+**************
+
+The ``message`` part of the previous step needs to be presented to the user. The user can sign it using one of the supported means. The following steps use the *dummy* means as example.
+
+Most of the means require the same steps:
+
+- start a session at the means backend
+- expose the session data to a user controlled device
+- poll the means backend of session updates
+- retrieve user signature upon success
+
+A session can be started by doing the following request:
+
+.. code-block::
+
+    POST /internal/auth/experimental/signature/session HTTP/1.1
+    Host: server.example.com
+    Content-Type: application/json
+
+    {
+      "means": "dummy",
+      "payload": "EN:PractitionerLogin:v3 I hereby declare to act on behalf of Nursing home A. This declaration is valid from Monday, 24 June 2019 14:32:00 until Monday, 24 June 2019 16:32:00."
+    }
+
+As you can see, the ``payload`` is the same as the ``message`` from the *drawup* step. The ``means`` must state the desired means. Next to the specified means in `RFC002 <https://nuts-foundation.gitbook.io/drafts/rfc/rfc002-authentication-token>`_, the Nuts OS implementation also supports a *dummy* means which always succeeds. The *dummy* means is not usable in *strict* mode.
+
+The result contains information on how to present a challenge to the user. In the case of the *dummy* means this is not needed. The result also contains a ``sessionID`` which can be used to poll the status and a ``sessionPtr`` which contains means specific information about the challenge.
```

Comment:
Now a subject for this PR, but I just noticed we mix casing for acronyms here (which triggers my OCD); `sessionID` and `sessionPtr`